crates/uv/src/lib.rs

```diff
@@ -1128,6 +1128,13 @@ async fn run(mut cli: Cli) -> Result<ExitStatus> {
                 warn_user_once!("`uv publish` is experimental and may change without warning");
             }
 
+            if args.skip_existing {
+                bail!(
+                    "`uv publish` does not support `--skip-existing`, \
+                    use `--check-url` with the simple index URL instead."
```

Comment:
Can you give users a little more context here?

Reply:
(I think this will be fairly common to encounter and it seems worth explaining why the API is different)

Reply:
Like what sort of context?

Reply:
Yeah like

> `uv publish` does not support `--skip-existing` because there is not a reliable way to identify when uploads fail due to existing distributions. Instead, use `--check-url` to provide the URL to the simple API for your index. uv will check the index for existing distributions before attempting uploads.